json/access/src/test/java/ch/qos/logback/contrib/json/access/JsonLayoutTest.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2014, The logback-contrib developers. All rights reserved.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation
+ *
+ *   or (per the licensee's choosing)
+ *
+ * under the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation.
+ */
+package ch.qos.logback.contrib.json.access;
+
+import ch.qos.logback.access.TeztConstants;
+import ch.qos.logback.access.dummy.DummyAccessEventBuilder;
+import ch.qos.logback.access.joran.JoranConfigurator;
+import ch.qos.logback.access.spi.AccessContext;
+import ch.qos.logback.access.spi.IAccessEvent;
+import ch.qos.logback.core.joran.spi.JoranException;
+import ch.qos.logback.core.read.ListAppender;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+public class JsonLayoutTest {
+
+    AccessContext context = new AccessContext();
+
+    void configure(String file) throws JoranException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback-contrib/file/2370841729/issues/source?bid=3222643&fileBranchId=3222645#l34)